.github/workflows/legacy-checkpoints.yml

```diff
@@ -80,8 +80,7 @@ jobs:
     - name: Install PL version
       run: |
         pip install "pytorch-lightning==${{ inputs.pl_version }}" \
-          -f https://download.pytorch.org/whl/cpu/torch_stable.html \
-          --extra-index-url https://test.pypi.org/simple/
+          -f https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
this is dropped as we observed that other dependencies could have corrupted versions of future versions here and we are not able to limit using test PyPI only for lightning package...